ecf/hurricane/plots/jevs_hurricane_global_det_tropcyc_plots.ecf

```diff
@@ -45,12 +45,12 @@ export cyc=00
 export job=jevs_hurricane_global_det_tropcyc_plots_${cyc}
 
 #Define the directory for TC-stats file 
-export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/1.0/hurricane_global_det/tropcyc/stats
+export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/v1.0/hurricane_global_det/tropcyc/stats
```

Comment:
Use ${evs_ver} as defined in `run.ver` in place of "v1.0" in the COMINstats definition.